samples/web/content/apprtc/js/appcontroller.js

```diff
@@ -288,10 +341,17 @@ AppController.prototype.onKeyPress_ = function(event) {
   }
 };
 
-AppController.prototype.displaySharingInfo_ = function(roomLink) {
+AppController.prototype.pushCallNavigation_ = function(roomId, roomLink) {
+  if (!isChromeApp()) {
+    window.history.pushState({'roomId': roomId, 'roomLink': roomLink }, roomId, roomLink);
+  }  
+};
+
+AppController.prototype.displaySharingInfo_ = function(roomId, roomLink) {
   this.roomLinkHref_.href = roomLink;
   this.roomLinkHref_.text = roomLink;
   this.roomLink_ = roomLink;
+  this.pushCallNavigation_(roomId, roomLink);
```

Comment:
We don't have to do it, but it makes the uri bar show the uri of the current room so that if they want to bookmark it, or copy/paste the direct connection uri they can. It also allows us to use forward/back to get back to the landing page screen.